services/amo/amo-downloads.service.js

```diff
@@ -12,15 +12,15 @@ class AmoWeeklyDownloads extends BaseAmoService {
   static category = 'downloads'
   static route = { base: 'amo/dw', pattern: ':addonId' }
 
-  static examples = [
-    {
-      title: 'Mozilla Add-on',
-      namedParams: { addonId: 'dustman' },
-      staticPreview: this.render({ downloads: 120 }),
-      keywords,
-      documentation,
+  static openApi = {
+    '/amo/dw/{addonId}': {
+      get: {
+        summary: 'Mozilla Add-on Downloads',
+        description: documentation,
```

Comment:
Another mega-nit, but do you think we should rename the `documentation` vars as part of these conversion efforts? I'm not partial to one name over the other, but I feel like the main, perhaps only, reason we chose `documentation` as the var name in the various service classes was because that's what the example objects needed

Reply:
Yeah agreed. Just calling the var `description` would be better. I'll probably do that next time I push more commits to the branch, but as you say, it is a style point, not a huge issue.

Reply:
Done in https://github.com/badges/shields/pull/9320/commits/b1de47f7cb85a4138cd6967af08a4de42189b06d

Reply:
I should've noted before this wasn't a blocker by any stretch, kinda a half question/suggestion, though I suspect it'll end up being a worthwhile change overall